### DISCUSSION TITLE
Django template does not exist error

### DISCUSSION THREAD

**Discussion by panchovilla1970** [GreaterWMS/GreaterWMS#273] [General]
Hi.. when I run daphne -p 8008 greaterwms.asgi:application I have the following error. Any idea? thanks

Template-loader postmortem
Django tried loading these templates, in this order:

Using engine django:

django.template.loaders.filesystem.Loader: C:\Python\GreaterWMS\templates\dist\spa\index.html (Source does not exist)
django.template.loaders.app_directories.Loader: c:\users\magda\appdata\local\programs\python\python39\lib\site-packages\django\contrib\admin\templates\dist\spa\index.html (Source does not exist)
django.template.loaders.app_directories.Loader: c:\users\magda\appdata\local\programs\python\python39\lib\site-packages\django\contrib\auth\templates\dist\spa\index.html (Source does not exist)
django.template.loaders.app_directories.Loader: c:\users\magda\appdata\local\programs\python\python39\lib\site-packages\rest_framework\templates\dist\spa\index.html (Source does not exist)
django.template.loaders.app_directories.Loader: c:\users\magda\appdata\local\programs\python\python39\lib\site-packages\django_filters\templates\dist\spa\index.html (Source does not exist)

and in the PowerShell

django.template.exceptions.TemplateDoesNotExist: dist/spa/index.html

**Comment by Singosgu** [comment]
in templates

quasar build
